README.md

```diff
@@ -30,20 +30,22 @@ For a quick introduction to Joinmarket you can watch [this demonstration](https:
 
 ### Quickstart - RECOMMENDED INSTALLATION METHOD (Linux and macOS only)
 
-Once you've downloaded this repo, either as a tar/zip file, and extracted it, or via `git clone`:
+Download the latest [release](https://github.com/Joinmarket-Org/joinmarket-clientserver/releases) as tar or zip, and extract it.
 
-Make sure to validate the signature on the tar/zip file provided on the [release page](https://github.com/Joinmarket-Org/joinmarket-clientserver/releases),
-or check the signature in git if you install that way using `git log --show-signature`.
+Make sure to validate the signature on the tar/zip file provided with the [release](https://github.com/Joinmarket-Org/joinmarket-clientserver/releases) (
+or check the signature in git if you install that way using `git log --show-signature`).
```

Comment:
```suggestion
Make sure to validate the signature on the tar/zip file provided with the [release](https://github.com/Joinmarket-Org/joinmarket-clientserver/releases)
(or check the signature in git if you install that way using `git log --show-signature`).
```
Extra leading space inside parenthesis.

Reply:
Thanks, done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1258/commits/42e64596ab9822421c4ce234327318b66e4413ea